### PR TITLE
stability: add fault-trigger server

### PR DESCRIPTION
**Pull request by cwen0**
1. stop / start  virtual machine 
2. stop / start etcd 
3. stop / start kubelet  

@weekface @zyguan @xiaojingchen PTAL

### CODE REVIEW

File: tests/pkg/fault-trigger/manager/vm.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+// Name       MAC address          Protocol     Address
+// -------------------------------------------------------------------------------
+// eth0       52:54:00:4c:5b:c0    ipv4         172.16.30.216/24
+// -          -                    ipv6         fe80::5054:ff:fe4c:5bc0/64
+// output: 172.16.30.216, nil
+func parserIP(data string) (string, error) {
+	data = stripEmpty(data)
+	lines := strings.Split(data, "\n")
+	for _, line := range lines {
+		if !strings.HasPrefix(line, "eth0") {
```
Comment (xiaojingchen) [inline tests/pkg/fault-trigger/manager/vm.go]: the interface name maybe not `eth0`.

Comment (cwen0) [reply tests/pkg/fault-trigger/manager/vm.go]: how about checking `ipv4` ?

Comment (xiaojingchen) [reply tests/pkg/fault-trigger/manager/vm.go]: check the `MAC address` is exist is more reasonable

Comment (cwen0) [reply tests/pkg/fault-trigger/manager/vm.go]: https://github.com/pingcap/tidb-operator/pull/312/files#diff-cc0457c3ced0a4de0bcef6986b537b8aR72  
if this check was failed, the program will check the `mac address`

File: tests/pkg/fault-trigger/manager/etcd.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import (
+	"os/exec"
+)
+
+// StartETCD starts etcd
+func (m *Manager) StartETCD() error {
+	shell := "systemctl start etcd"
+	cmd := exec.Command("/bin/sh", "-c", shell)
+	_, err := cmd.CombinedOutput()
```
Comment (weekface) [inline tests/pkg/fault-trigger/manager/etcd.go]:
```
data, err := cmd.CombinedOutput()
if err != nil {
    log.Errorf(".............")
}
```
?

**Comment by cwen0** [comment]
@weekface @xiaojingchen  PTAL again, thanks!

**Pull request merged**